mapflow/functional/service/data_catalog.py

```diff
@@ -204,6 +204,12 @@ def upload_images(self,
         else:
             image_to_upload = image_paths[0]
             non_uploaded = image_paths[1:]
+            # When first failed image occures, stop uploading if mosaic was deleted
```

Comment:
What if it is not the first error?

I would prefer this, to check only for error 404 (which certainly is not a recoverable error).

```
if failed and 404 == response.status_code:
    ....
```

Reply:
Yes, but for QNetworkReply it's `response.error() == 203` (ContentNotFoundError)